corehq/apps/cleanup/migrations/0012_add_es_index_to_checkpoint_ids.py

```diff
@@ -59,6 +59,7 @@ class Migration(migrations.Migration):
 
     dependencies = [
         ('cleanup', '0011_merge_couch_sql_pillows'),
+        ('pillowtop', '0003_auto_20170411_1957'),
```

Comment:
I think we could actually no-op migrations in the cleanup directory after they've been run, but will leave that for another day